core/eolearn/core/eonode.py

```diff
@@ -42,7 +42,7 @@ def __post_init__(self):
         if self.name is None:
             super().__setattr__('name', self.task.__class__.__name__)
 
-        super().__setattr__('uid', generate_uid(self.name))
+        super().__setattr__('uid', generate_uid(self.task.__class__.__name__))
```

Comment:
This ignores custom node names. IMO it would be better to use `self.get_custom_name()`

Reply:
With task class name it looks much better in reports where we then write e.g.:

```
Load features (LoadTask-c2cabaa677a311ec8d2e-ae4c34840284) 
```

(We show both custom name and uid.)

Reply:
fine, let's leave it at that for now. Not likely to cause issues i hope